main.go

```diff
@@ -10,6 +10,7 @@ import (
 	"net/http"
 	"os"
 	"path/filepath"
+	"regexp"
 	"strconv"
 	"strings"
 	"text/tabwriter"
```

Comment:
I know it's not your code modification, but could you please remove the alias testing from the imports, as it is unneeded.